code/datums/uplink_items/uplink_general.dm

```diff
@@ -664,6 +664,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/implanter/adrenalin
 	cost = 8
 
+/datum/uplink_item/implants/stealthimplant
+	name = "Stealth Bio-chip"
+	desc = "This one-of-a-kind implant will make you almost invisible if you play your cards right. \
+			On activation, it will conceal you inside a chameleon cardboard box that is only revealed once someone bumps into it."
+	reference = "SI"
```

Comment:
this will need to change soon with edan's PR I believe (unless that was just wiz entries?)

Reply:
That was just wiz entires (though these could do with a similar same treatment)